source/mite.js

```diff
@@ -27,6 +27,7 @@ program
   .command('stop', 'stop any running counter')
   .command('unlock', 'unlock single time entry')
   .command('users', 'list, filter & search for users')
+  .command('clearcache', 'clear cache of mite data')
```

Comment:
I’d rather go for just "cache" with a sub-command "clear" or "remove". This leaves it open for other commands like "renew" or "update".

Reply:
You're right, I changed command name and description